server/index.js

```diff
@@ -35,6 +35,41 @@ app.get('/api/hangouts', (req, res, next) => {
     });
 });
 
+app.get('/api/hangouts/:hangoutId', (req, res, next) => {
+  const parse = parseInt(req.params.hangoutId);
```

Comment:
Work on better variable names in the future! Maybe something like parsedHangoutId? Something more verbose so it's very clear what it is.